ios/RNDFPBannerView.m

```diff
@@ -77,6 +80,17 @@ - (void)setTestDevices:(NSArray *)testDevices
     _testDevices = RNAdMobProcessTestDevices(testDevices, kDFPSimulatorID);
 }
 
+- (void)setCustomTargeting:(NSDictionary *)customTargeting
+{
+    if(![customTargeting isEqual:_customTargeting]) {
+        _customTargeting = customTargeting;
+        if (_bannerView) {
```

Comment:
@Almouro This block causes the request to be made twice on initial load - did we add it originally so that we had the ability to change props on the Banner View?  This behaviour is inconsistent with the other props on this view (which don't have this behaviour).

Looks like the guys at NFL decided that it caused a bug for them too: https://github.com/nfl/react-native-admob/pull/3

Can we remove this block, or was there a reason we added it originally?